packages/flutter/lib/src/services/semantics_service.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'system_channels.dart';
+
+/// Allows access to the semantic feedback interface on the device.
+class SemanticsService {
+  SemanticsService._();
+
+  /// Sends a semantic announcement.
+  ///
+  /// Can be used for on-demand accessibility announcements.
```

Comment:
Please add more details here, such as what is suitable to be sent in this manner, when you would want to use it, when you would want to avoid it, etc.

Reply:
Also do we need to send the locale and/or text direction of the text being sent?

Reply:
Done (doc improvement).

Android's announceForAccessibility does not take locale or text direction.
And same for iOS UIAccessibilityAnnouncementNotification as far as I can tell.

Reply:
Following f2f - added textDirection as we might want to use it in Flutter even though iOS/Android are not using it for these announcements.